exercises/list-ops/canonical-data.json

```diff
@@ -57,6 +57,14 @@
             "lists": [[1, 2], [3], [], [4, 5, 6]]
           },
           "expected": [1, 2, 3, 4, 5, 6]
+        },
+        {
+          "description": "list of nested lists",
+          "property": "concat",
+          "input": {
+            "lists": [[[1, 2]], [[3]], [[]], [[4, 5, 6]]]
```

Comment:
Initial reaction: It would have been more instructive to display some necessary merging, perhaps for example:

```
[[[1], [2]], [[3]], [[]], [[4, 5, 6]]]
```

(with the same expected result)

Upon review, I don't think these two differ meaningfully in what implementations they will accept/reject, so I should not request such a change.

Reply:
Wouldn't the output of this concat be `[[1], [2], [3], [], [4, 5, 6]]` and not `[[1, 2], [3], [], [4, 5, 6]]`? Or perhaps I'm misunderstanding something. :thinking:
I do like this suggestion, if I understand it correctly, because the number of elements from input to output change, whereas in the current test it does not.

Reply:
You got it, I was wrong. Output of `[[[1], [2]], [[3]], [[]], [[4, 5, 6]]]` is your suggested `[[1], [2], [3], [], [4, 5, 6]]` instead of my bogus suggestion.

Reply:
I like this idea and it should also be compatible with any languages that require all elements of a list be of the same type. Pushing now!